### DISCUSSION TITLE
Turborepo deployment issue with the examples/basic

### DISCUSSION THREAD

**Discussion by Saad1379** [vercel/turborepo#8887] [Help] [answered]
### Summary

I have tried to deploy the example turborepo project found here

https://vercel.com/templates/next.js/monorepo-turborepo
https://github.com/vercel/turbo/tree/main/examples/basic

But the build process fails with the following error
![image](https://github.com/user-attachments/assets/b87c2517-de8c-42f7-a4d1-1f47d9041643)

**To Reproduce Issue**
1. Login to your vercel account.
2. Open the link https://vercel.com/templates/next.js/monorepo-turborepo
3. Click Deploy

### Additional information

```js
Error Details:

web:build: └ ○ /_not-found                          895 B          89.8 kB
web:build: + First Load JS shared by all            88.9 kB
web:build:   ├ chunks/207a5ba7-ac24752941b7dbc9.js  51.5 kB
web:build:   ├ chunks/479-8568a017162d290a.js       35.5 kB
web:build:   └ other shared chunks (total)          1.88 kB
web:build: 
web:build: 
web:build: ○  (Static)  prerendered as static content
web:build: 
  Tasks:    2 successful, 2 total
 Cached:    2 cached, 2 total
   Time:    579ms >>> FULL TURBO
Summary:    /vercel/path0/.turbo/runs/2jz7gXxvGd5kjGRcRXf04TfCir1.json
Error: No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your Project Settings.
Learn More: https://vercel.link/missing-public-directory
```

### Example

https://vercel.com/templates/next.js/monorepo-turborepo

**Comment by john093e** [comment]
Same +1 with another turborepo

**Reply by Saad1379** [reply, marked as answer]
I resolved it by setting root directory to apps/[my_site]. In my case I set the root directory in Vercel Project settings to "apps/web" and it worked

**Reply by john093e** [reply]
As for my case I was trying to deploy a full turborepo without pointing the app. sorry false alarm :)

**Comment by anthonyshew** [comment]
This template is now fixed! For future reference, checking that you've set your Root Directory is usually the fix for this, like in [your comment](https://github.com/vercel/turbo/discussions/8887#discussioncomment-10231801).
